src/screens/CreateCommunity/index.js

```diff
@@ -32,6 +32,7 @@ const CreateCommunity = () => {
   const [users, dispatch] = React.useContext(Context).users;
   const [name, setName] = React.useState('');
   const [typeFetch, setTypeFetch] = React.useState('');
+  const inputRef = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `useRef` hook is a good practice for managing focus on an input element. This allows you to directly interact with the DOM node in functional components where refs are not readily available.
```javascript
const inputRef = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->